octobot/src/server/github_handler.rs

```diff
@@ -406,6 +407,25 @@ impl GithubEventHandler {
         let mut participants = pull_request.assignees();
         // add the author of the PR
         participants.push(pull_request.user().clone());
+        // add team members
+        let teams = pull_request.teams();
+        let futures = teams.iter().map(|t| {
+            self.github_session
+                .get_team_members(t.org(), t.slug.as_str())
+        });
+        let teams = join_all(futures).await;
```

Comment:
imo, it's nice to make this `future::join_all`, and for the import statement just have `use futures::future` -- that way it's clearer where the function is coming from.